build/Dockerfile.tz

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+RUN microdnf clean all
```

Comment:
Is it necessarry to run `clean all` and `update` ? If so, we should run all those 3 `microdnf` commands as one `RUN` to avoid creating another layers in image.

Reply:
Here is the error I am facing when I run the microdnf install 
Downloading metadata...
error: cannot update repo 'ubi-8-baseos': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried; Last error: Curl error (60): Peer certificate cannot be authenticated with given CA certificates for https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/x86_64/baseos/os/repodata/repomd.xml [SSL certificate problem: self signed certificate in certificate chain]

Reply:
Are you sure that `tzdata`  is missing in the image?

Reply:
when I try running tzselect on the keda-operator pod (exec), I don't see the directory /usr/share/zoneinfo

/usr/bin/tzselect: line 180: /usr/share/zoneinfo/iso3166.tab: No such file or directory
 /usr/bin/tzselect: time zone files are not set up correctly